openstack/compute/v2/extensions/services/requests.go

```diff
@@ -11,3 +11,34 @@ func List(client *gophercloud.ServiceClient) pagination.Pager {
 		return ServicePage{pagination.SinglePageBase(r)}
 	})
 }
+
+// UpdateOpts specifies the base attributes that may be updated on a service.
+type UpdateOpts struct {
+	// Status represents the new service status. One of enabled or disabled.
+	Status string `json:"status,omitempty"`
+
+	// DisabledReason represents the reason for disabling a service.
+	DisabledReason string `json:"disabled_reason,omitempty"`
+
+	// ForcedDown is a manual override to tell nova that the service in question
+	// has been fenced manually by the operations team.
+	ForcedDown bool `json:"forced_down,omitempty"`
```

Comment:
The combination of `bool` and `omitempty` will cause values of false to never be sent. Just to be sure: there is absolutely no reason to ever send this field with a value of false, correct?

Reply:
Correct, that is my understanding.